components/PrimaryButton.tsx

```diff
@@ -19,6 +21,7 @@ const PrimaryButton = ({
   linkTo,
   isDisable,
   width,
+  whiteButton,
```

Comment:
Instead of adding a whiteButton prop it would be better to have customCss prop here and then you can pass any css you want from here

Reply:
done